src/apps/js_generic/js_generic_base.cpp

```diff
@@ -527,16 +511,11 @@ namespace ccfapp
       extensions.emplace_back(
         std::make_shared<ccf::js::extensions::HistoricalExtension>(
           &context.get_historical_state()));
-      // add openenclave.*
-      extensions.emplace_back(
-        std::make_shared<ccf::js::extensions::OpenEnclaveExtension>());
-      // add snp_attestation.*
-      extensions.emplace_back(
-        std::make_shared<ccf::js::extensions::SnpAttestationExtension>());
 
       interpreter_cache->set_interpreter_factory(
         [extensions](ccf::js::TxAccess access) {
-          auto interpreter = std::make_shared<js::core::Context>(access);
+          // CommonContext also adds many extensions
```

Comment:
This is true, but not very informative perhaps.

Reply:
There's comments within `CommonContext` listing _what_ is added, that I don't want to duplicate. This is intended as a pointer down the behaviour chain - if you have extensions that aren't listed _here_, they're likely _there_.